lib/task/import/csvCheckImportTask.class.php

```diff
@@ -189,6 +215,43 @@ public function execute($arguments = array(), $options = array())
       }
     }
 
+    // Check that parent IDs actually exist if legacy and parent IDs were found
+    if (count($legacyIds) && count($parentIds))
```

Comment:
I think the checker should report **all** `parentIds` as missing if `0 == count(legacyIds)`.